python/taichi/profiler/kernelprofiler.py

```diff
@@ -61,42 +70,137 @@ def set_kernel_profiler_mode(self, mode=False):
                             f'is not supported')
 
     def get_kernel_profiler_mode(self):
+        """Get status of :class:`~taichi.profiler.kernelprofiler.KernelProfiler`."""
         return self._profiling_mode
 
-    def clear_frontend(self):
-        self._total_time_ms = 0.0
-        self._traced_records.clear()
-        self._statistical_results.clear()
-
     def get_total_time(self):
-        self.update_records()  # traced records
-        self.count_results()  # _total_time_ms is counted here
+        """Get elapsed time of all kernels recorded in KernelProfiler.
+
+        Returns:
+            time (float): total time in second.
+        """
+        self._update_records()  # kernel records
+        self._count_statistics()  # _total_time_ms is counted here
         return self._total_time_ms / 1000  # ms to s
 
+    def clear_info(self):
+        """Clear all records both in front-end :class:`~taichi.profiler.kernelprofiler.KernelProfiler` and back-end instance ``KernelProfilerBase``.
+
+        Note:
+        The values of ``self._profiling_mode`` and ``self._metric_list`` will not be cleared.
+        Set ``self._profiling_mode`` via :func:`~taichi.profiler.kernelprofiler.KernelProfiler.set_kernel_profiler_mode`.
+        Set ``self._metric_list`` via :func:`~taichi.profiler.kernelprofiler.KernelProfiler.set_metrics`.
+        """
+        #sync first
```

Comment:
nit: Such comments do not provide more info, we can infer the behavior directly by reading the code...